src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,101 @@
+package parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+import org.junit.jupiter.api.Test;
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.InvalidCommand;
+import command.ListCommand;
+import command.MarkCommand;
+import command.UnmarkCommand;
+import task.Deadline;
+import task.Event;
+import task.ToDo;
+
+public class ParserTest {
+    @Test
+    public void testParseValidListCommand() {
```

Comment:
A very minor point. But maybe to be safe you could follow the naming convention for testing functions here: https://se-education.org/se-book/cppToJava/junit/basic/index.html